CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+* release/v1.x, (for example [release/v1.5](https://github.com/rancher/rke/tree/release/v1.5)
+* release/v1.(x-1) (for example [release/v1.4](https://github.com/rancher/rke/tree/release/v1.4)
+
+Each track releases an RKE binary. Each release of RKE officially supports a few Kubernetes versions. Some versions are shared by both tracks but normally the track with the higher minor version includes at least one newer Kubernetes version. For example, as it can be observed in [release v1.4.11](https://github.com/rancher/rke/releases/tag/v1.4.11), the following Kubernetes versions are supported:
+* v1.26.9-rancher1-1
+* v1.25.14-rancher1-1
+* v1.24.17-rancher1-1
+* v1.23.16-rancher2-3
+
+whereas [release v1.5.0](https://github.com/rancher/rke/releases/tag/v1.5.0) includes:
+* v1.25.14-rancher1-1
+* v1.26.9-rancher1-1
+* v1.27.6-rancher1-1
+
+If a Kubernetes version is supported, that means we can deploy a cluster with that Kubernetes version using the binary included in that RKE release.
+
+When doing a commit in RKE code, we must carefully think if we want that commit in both tracks and hence backported. Sometimes, there is a reason to no backport it, for example a flag, that we would like to limit to only the newer track. However, that is exceptional and normally commits end up in both branches as a way to reduce the differences between those and simplify RKE maintainability. Therefore, usually we backport a commit between both RKE tracks. In that case, the following steps should be applied:
+
+* Create an Issue in rancher/rancher github project for Project Managers and QA to track the commit. Set the decided milestone for release/v1.x in the issue
+* Create another Issue in the same github project and in the description, specify it is a backport of the previous commit. Set the decided milestone for release/v1.(x-1) in the issue
+* Create the PR with the commit in branch release/v1.x and in the description, link the newly created issue
+* Cherry-pick the commit in branch release/v1.(x-1) and in the description, link the created issue for that RKE track.
+
+A commit is normally available in all supported Kubernetes versions of the release unless scoped in the code to a specific version. Here are two examples for this:
+* Available in all supported Kubernetes versions: https://github.com/rancher/rke/commit/1dad2582e1efe5564b6d1a7385afad70d5dbd1f7
+* Scoped to a specific version: https://github.com/rancher/rke/commit/feca6a3048c1f8c9031ab2849a46c46a7ea1d673
+
+Note that in the scoped case, there is a check on the version, which is normally fetch by executing the line `util.StrToSemVer(k8sVersion)`, [here an example](https://github.com/rancher/rke/blob/release/v1.5/cluster/validation.go#L752). In this case, as the same code is executed for all Kubernetes versions, unless scoped, there is no need to create extra PRs to effectively backport the code to all Kubernetes versions.
```

Comment:
> In this case, as the same code is executed for all Kubernetes versions, unless scoped, there is no need to create extra PRs to effectively backport the code to all Kubernetes versions. 

This line needs to be modified, backporting for rke1 is decided based on rke1 versions and once backported - it'll apply to k8s versions it supports. Branches aren't based on k8s versions.

Reply:
I was trying to explain that there is no need to backport stuff to different k8s versions (as we do in rke2/k3s) because the PR will apply to k8s versions supported by the rke1 version. Let me rephrase it a bit better